pkg/cmd/start/main.go

```diff
@@ -89,6 +89,9 @@ func NewStartCommand() *cobra.Command {
 	cmd.Flags().Int32("cr-metrics-port", 8686, "The metrics port for Operator and/or Custom Resource based metrics")
 	viper.BindPFlag("cr-metrics-port", cmd.Flags().Lookup("cr-metrics-port"))
 
+	cmd.Flags().String("documentation-url", "https://www.jaegertracing.io/docs/latest", "The URL for the 'Documentation' menu item")
```

Comment:
Not sure whether to make the default URL use the version, rather than latest?

Reply:
I think it's a great idea to tie it to the version.